src/index.html

```diff
@@ -9,10 +9,54 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The stylesheet linked here is `index.scss`. Ensure that this SCSS file reflects the calendar starting day as Sunday, as indicated by the class `calendar--start-day-sun` in the HTML. This alignment is crucial to maintain consistency in the calendar's appearance.